vendor/luautf8/build-m1.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+echo "Building for M1..."
```

Comment:
thanks for adding support for apple silicon .. maybe its better to call it apple silicon instead of M1, pretty sure there will be a m2 or m1x or something soon :D